src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -927,7 +927,9 @@ def create_vm(cmd, vm_name, resource_group_name, image=None, size='Standard_DS1_
                            'The "UbuntuLTS" will be removed with the breaking change release of Fall 2023.')
         if image in ["RHEL", "Debian", "CentOS", "Flatcar"]:
             logger.warning('Consider using the image alias including the version of the distribution you want to use. '
-                           'For example: please use Debian11 instead of Debian')
+                           'For example: please use Debian11 instead of Debian.\nIn Ignite (November) 2023, '
+                           'the aliases without version suffix (such as: `UbuntuLTS`, `CentOS`, `Debian`, `Flatcar`, '
+                           'and `RHEL`) will be removed.')
```

Comment:
add the aliases: openSUSE-Leap and SLES. 
On further review, those need to be removed as well as they are non-versioned. 
https://github.com/Azure/azure-rest-api-specs/pull/24149 will add the new openSUSE-Leap versioned alias. It looks odd to have 2 images for the same versions, but I think this is a good idea to allow users to migrate to the new alias months ahead of time. 

In PS, I was able to remove those aliases already because they were using old versions that no longer existed so they were already broken. So they weren't in the PS breaking message.

Reply:
Hi @Sandido, thank you for your message, I have already completed the above modification request. Please help review the PR again.

Reply:
> It looks odd to have 2 images for the same versions, but I think this is a good idea to allow users to migrate to the new alias months ahead of time.

@sles-15-sp3 At present, the `offer` corresponding to `openSUSE-Leap` *(offer: `opensuse-leap-15-3`)* and `SLES` *(offer: `sles-15-sp3`)* is different. Is it reasonable for them to point to the same image alias `SuseSles15SP3` *(offer: `sles-15-sp3`)* in the future?

Reply:
@zhoxing-ms , I don't understand. 
 "openSUSE-Leap": {
            "publisher": "SUSE",
            "offer": "opensuse-leap-15-3",
            "sku": "gen2",
            "version": "latest",
            "architecture": "x64"
          },

versus the new

"OpenSuseLeap154Gen2":  {
            "publisher":  "SUSE",
            "offer":  "openSUSE-leap-15-4",
            "sku":  "gen2",
            "version":  "latest",
            "architecture": "x64"
          },

and for SLES it is 
"SLES": {
            "publisher": "SUSE",
            "offer": "sles-15-sp3",
            "sku": "gen2",
            "version": "latest",
            "architecture": "x64"
          },

Then for the new versioned SLES it is
"SuseSles15SP3": {
            "publisher": "SUSE",
            "offer": "sles-15-sp3",
            "sku": "gen2",
            "version": "latest",
            "architecture": "x64"
          },

as seen here: https://github.com/Azure/azure-rest-api-specs/pull/24149/files

Reply:
@Sandido Because I didn't notice the alias `OpenSuseLeap154Gen2` before, so I misunderstood your meaning. Thanks for your clarification